lib/helpers.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * The argument *must* be named `contract`
```

Comment:
I did try to instead wrap it as eg
```
((c) => ...)(contract)
```
so that it would be possible to name the argument anything but it just silently did nothing - I assume static-eval was not happy with the wrapping